versioned_docs/version-0.4.2/tutorials/react-native.mdx

```diff
@@ -124,7 +152,7 @@ Run `pod install` in the /ios directory to install the new pods
 
 ### Native permissions configuration
 
-In order for camera and audio to work you'll need to add some native configuration:
+For this tutorial in order for camera and audio to work you'll need to add some native configuration:
 
 You need to at least set up camera permissions.
 
```

Comment:
`You can edit this file in Xcode` a couple of lines below introduces confusion, the editor in this step doesn't matter

Reply:
Also, in "On Android add to your `AndroidManifest.xml`" there should be mentioned the tag where to put these permission (a mobile bambik like me put it under internet permission in the root `manifest` tag")

Reply:
That is true, it can be do in any text editor. For some people it is just easier to do it in xcode because you can add it dropdown list.